lib/ddtrace/contrib/rails/action_controller.rb

```diff
@@ -51,15 +51,17 @@ def self.process_action(_name, start, finish, _id, payload)
             if payload[:exception].nil?
               # [christian] in some cases :status is not defined,
               # rather than firing an error, simply acknowledge we don't know it.
-              span.set_tag(Datadog::Ext::HTTP::STATUS_CODE, payload.fetch(:status, '?').to_s)
+              status = payload.fetch(:status, '?').to_s
+              span.status = 1 if status.starts_with?('50')
+              span.set_tag(Datadog::Ext::HTTP::STATUS_CODE, status)
```

Comment:
is it possible for us to include a stacktrace in either of these clauses? This is consistent with what we do in span.set_error https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/span.rb#L89